### PR TITLE
fixed Try .NET link in Readme

### PR DESCRIPTION
**Pull request by morganwm**
link for Try .Net seemed to be pointing to incorrect docs link

### CODE REVIEW

**Comment by morganwm** [comment]
Replace link in "What is the Repo for" section with correct try.dot.net link

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Try .NET is an inline code runner and playground that enables people to experime
 **Note**: Currently,Try .NET only supports C# but we do plan to enable support F# and VB in the near future. 
 
 ### What is the repo for?
-- A place where you can post issues about the [Try .NET](https://docs.microsoft.com/en-us/dotnet/api/system.datetime?view=netframework-4.7.2#invoke-constructors).
+- A place where you can post issues about the [Try .NET](https://try.dot.net/).
```
Comment (jonsequitur) [inline README.md]: Thanks for noticing this. 

The URL should probably be https://dotnet.microsoft.com/platform/try-dotnet.

Also, would you mind removing the word "the"?

**Comment by morganwm** [comment]
Removed the word "the" and updated the link to reflect the full landing page link URL

File: README.md (hunk 1 above)
Comment (morganwm) [reply README.md]: No worries, I have made the requested changes!

Comment (jonsequitur) [reply README.md]: Thanks!

**Pull request merged**